lametro/views.py

```diff
@@ -80,9 +80,11 @@ def get_context_data(self, **kwargs):
         except:
             context['packet_url'] = None
 
-        # Create list of related board reports
+        # Create list of related board reports, ordered by descending last_action_date.
         if context['legislation'].related_bills.all():
-            context['related_bills'] = [LAMetroBill.objects.get(identifier=bill.related_bill_identifier) for bill in context['legislation'].related_bills.all()]
+            related_bills = [LAMetroBill.objects.get(identifier=bill.related_bill_identifier) for bill in context['legislation'].related_bills.all()]
+            context['related_bills'] = sorted(related_bills, key=lambda bill: bill.last_action_date, reverse=True)
```

Comment:
nice work. ;-) the sorted method is a great approach here, and for the audio links as well, because we know the thing we're sorting is going to be relatively small. a word to the wise (i learned this the hard way recently): if you're working with a larger thing, sorting can be suuuper inefficient.

Reply:
what have you implemented instead in those cases?